source/locale/pt_BR/LC_MESSAGES/using-web-interface.po

```diff
@@ -8,27 +8,23 @@ msgstr ""
 "Project-Id-Version: dojot 0.2.0\n"
 "Report-Msgid-Bugs-To: \n"
 "POT-Creation-Date: 2020-10-20 15:54-0300\n"
-"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
-"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
-"Language-Team: LANGUAGE <LL@li.org>\n"
+"PO-Revision-Date: 2020-12-09 11:16-0300\n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=utf-8\n"
 "Content-Transfer-Encoding: 8bit\n"
 "Generated-By: Babel 2.8.0\n"
+"Last-Translator: \n"
+"Language-Team: \n"
+"Language: pt_BR\n"
+"X-Generator: Poedit 2.4.2\n"
 
 #: ../../source/using-web-interface.rst:2
 msgid "Using web interface"
 msgstr "Usando a interface WEB"
 
 #: ../../source/using-web-interface.rst:4
-msgid ""
```

Comment:
It seems to have changes in the `using-web-interface.po` translation file, but not in its rst

Reply:
done

Reply:
Are there any changes to the using-web-interface.rst file?

Reply:
Yes, are updates. The PO file have the correction bellow:

#: ../../source/using-web-interface.rst:4
msgid "This tutorial will show how to do basic operations in dojot, such as creating devices, checking its attributes and creating flows, import/export and firmware update."
msgstr "Este tutorial descreve as operações básicas na dojot, como criar dispositivos, conferir seus atributos e criar fluxos, importar/exportar e atualizar firmware."

Reply:
But does it really have anything to fix in using-web-interface.rst?

Reply:
I updated the RST references and I believe that solving the problem

Reply:
My question is, has nothing to be corrected in using web interface in English, only in Portuguese?

Reply:
I fixed the RST and PO files according the suggestions.